docs/blog/selectors.md

```diff
@@ -0,0 +1,386 @@
+│ Adelie  │ Torgersen │      38.900002 │     17.799999 │             181.0 │      3625.0 │ female │  2007 │
+│ Adelie  │ Torgersen │      39.200001 │     19.600000 │             195.0 │      4675.0 │ male   │  2007 │
+│ Adelie  │ Torgersen │      34.099998 │     18.100000 │             193.0 │      3475.0 │ ∅      │  2007 │
+│ Adelie  │ Torgersen │      42.000000 │     20.200001 │             190.0 │      4250.0 │ ∅      │  2007 │
+│ …       │ …         │              … │             … │                 … │           … │ …      │     … │
+└─────────┴───────────┴────────────────┴───────────────┴───────────────────┴─────────────┴────────┴───────┘
+```
+
+We can make all string columns have the same case too!
+
+```
+In [35]: t.mutate(s.across(s.of_type("string"), _.lower()))
+Out[35]:
+┏━━━━━━━━━┳━━━━━━━━━━━┳━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━┳━━━━━━━━┳━━━━━━━┓
+┃ species ┃ island    ┃ bill_length_mm ┃ bill_depth_mm ┃ flipper_length_mm ┃ body_mass_g ┃ sex    ┃ year  ┃
+┡━━━━━━━━━╇━━━━━━━━━━━╇━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━╇━━━━━━━━╇━━━━━━━┩
+│ string  │ string    │ float64        │ float64       │ int64             │ int64       │ string │ int64 │
+├─────────┼───────────┼────────────────┼───────────────┼───────────────────┼─────────────┼────────┼───────┤
+│ adelie  │ torgersen │           39.1 │          18.7 │               181 │        3750 │ male   │  2007 │
+│ adelie  │ torgersen │           39.5 │          17.4 │               186 │        3800 │ female │  2007 │
+│ adelie  │ torgersen │           40.3 │          18.0 │               195 │        3250 │ female │  2007 │
+│ adelie  │ torgersen │            nan │           nan │                 ∅ │           ∅ │ ∅      │  2007 │
+│ adelie  │ torgersen │           36.7 │          19.3 │               193 │        3450 │ female │  2007 │
+│ adelie  │ torgersen │           39.3 │          20.6 │               190 │        3650 │ male   │  2007 │
+│ adelie  │ torgersen │           38.9 │          17.8 │               181 │        3625 │ female │  2007 │
+│ adelie  │ torgersen │           39.2 │          19.6 │               195 │        4675 │ male   │  2007 │
+│ adelie  │ torgersen │           34.1 │          18.1 │               193 │        3475 │ ∅      │  2007 │
+│ adelie  │ torgersen │           42.0 │          20.2 │               190 │        4250 │ ∅      │  2007 │
+│ …       │ …         │              … │             … │                 … │           … │ …      │     … │
+└─────────┴───────────┴────────────────┴───────────────┴───────────────────┴─────────────┴────────┴───────┘
+```
+
+### Multiple Computations per Column
+
+What if I want to compute multiple things? Heck yeah!
+
+```
+In [26]: t.group_by("sex").mutate(
+    ...:     s.across(s.numeric() & ~s.c("year"), dict(cent=_ - _.mean(), zscore=(_ - _.mean()) / _.std()))
```

Comment:
```suggestion
    ...:     s.across(s.numeric() & ~s.c("year"), dict(center=_ - _.mean(), zscore=(_ - _.mean()) / _.std()))
```

Reply:
Hmm, I guess below you call this `centered` -- either is good, but nice to be consistent.

Reply:
I did this this cut down the width of the table a bit. I'll change the other to `cent`.

Reply:
Went with `centered` and just used another selector to subset some columns :)